docker/package/tezos_baking_wizard.py

```diff
@@ -132,6 +132,33 @@ def list_connected_ledgers():
     return [name.decode() for name in re.findall(ledger_regex, output.stdout)]
 
 
+def get_node_rpc_addr(network):
+    try:
+        with open("/lib/systemd/system/tezos-node-" + network + ".service", "r") as f:
```

Comment:
Let's get the path to the service file via `systemctl`, because the service can be modified manually via `systemctl edit` and the modified version is store in `/etc/systemd/system/`, in order to get path from the `systemctl`, you can use
`systemctl show -p FragmentPath tezos-node-<network>.service | cut -d'=' -f2`, or parse `systemctl show tezos-node-<network>.service` in code

Reply:
`systemctl show` also readily shows `Environment` with all the needed variables :thinking: Is it not worth checking them straight from that output? Checking them from the file would be marginally easier, sure, but (to the best of my knowledge) `systemctl edit` could be used to override the `Environment` directly, right?

Reply:
oh, indeed, it makes sense to parse `systemctl show` instead if `Environment` is presented in it. AFAIU, `systemctl show` will handle all overrides that happened to the service